modelci/app/v1/endpoints/model.py

```diff
@@ -78,11 +87,11 @@ def update_model_structure_as_new(id: str, structure, dry_run: bool = False):  #
             [A] fc1 -> fc2
 
         >>>
-        ... structure = {
+        ... structure_data = {
         ...     'layer': {
         ...         'fc': {'_op': 'D'},
```

Comment:
`_op` should be change to `op_` in this example

Reply:
Updated